test/documentation/DoxygenLayout.xml

```diff
@@ -0,0 +1,206 @@
+<doxygenlayout version="1.0">
+  <!-- Generated by doxygen 1.8.14 -->
+  <!-- Navigation index tabs for HTML output -->
+  <navindex>
+    <tab type="mainpage" visible="yes" title=""/>
+<!--     <tab type="pages" visible="yes" title="" intro=""/> -->
+
+    <tab type="user" visible="yes" title="Setup" url="@ref setup" intro=""/>
+    <tab type="usergroup" visible="yes" title="Tutorial" intro="">
+        <tab type="user"  visible="yes" title="A first example" url="@ref tutorial_first_example" intro=""/>
+        <tab type="user"  visible="yes" title="Alphabets"       url="@ref tutorial_alphabets"     intro=""/>
+    </tab>
+    <tab type="usergroup" visible="yes" title="Notes" intro="">
+        <tab type="user" visible="yes" title="API stability" url="@ref notes_stability" intro=""/>
+        <tab type="user" visible="yes" title="Serialisation" url="@ref notes_serialisation" intro=""/>
+    </tab>
+    <tab type="modules" visible="yes" title="API Reference (Modules)" intro=""/>
+    <tab type="usergroup" visible="yes" title="API Reference (details)" intro="">
+        <tab type="namespaces" visible="yes" title="">
+            <tab type="namespacelist" visible="yes" title="" intro=""/>
+            <tab type="namespacemembers" visible="yes" title="" intro=""/>
+        </tab>
+        <tab type="classes" visible="yes" title="">
+            <tab type="classlist" visible="yes" title="" intro=""/>
+            <tab type="classindex" visible="$ALPHABETICAL_INDEX" title=""/>
+            <tab type="hierarchy" visible="yes" title="" intro=""/>
+            <tab type="classmembers" visible="yes" title="" intro=""/>
+        </tab>
+        <tab type="files" visible="yes" title="">
+            <tab type="filelist" visible="yes" title="" intro=""/>
+            <tab type="globals" visible="yes" title="" intro=""/>
+        </tab>
+        <tab type="examples" visible="yes" title="" intro=""/>
+    </tab>
+  </navindex>
+
+  <!-- Layout definition for a class page -->
+  <class>
+    <briefdescription visible="yes"/>
+    <includes visible="$SHOW_INCLUDE_FILES"/>
+    <inheritancegraph visible="$CLASS_GRAPH"/>
+    <collaborationgraph visible="$COLLABORATION_GRAPH"/>
+    <memberdecl>
+      <nestedclasses visible="yes" title=""/>
```

Comment:
Well if we are already so specific about tabs, why are these not indented by 4 spaces?

Reply:
This was auto-generated by doxygen, but I can change it...

Reply:
Well, if this was only for an initial setup, I suggest making the indentation 4 spaces, yes.

Reply:
I already did